docker/candlepin-base/cp-test

```diff
@@ -429,6 +431,14 @@ if [ "$RSPEC" == "1" ]; then
     echo "Running rspec tests..."
     CLEAN_CP=1
 
+    # To run the cloud auth rspec test suite.
+    # We generate custom.yaml to enable the
+    # cloud auth property.
+    # This is valid only for hosted mode.
+    if [ "$CLOUD_AUTH_RSPEC" == "1" ] && [ "$HOSTED" == "1" ]; then
+      python jenkins/generate_custom_yaml.py
```

Comment:
The script should probably change location; The `jenkins` folder isn't appropriate since this is not a jenkins-specific script. `candlepin-base` or `server/bin`  is better